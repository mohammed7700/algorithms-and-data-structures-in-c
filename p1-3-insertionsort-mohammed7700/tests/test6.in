17
23
4
98
99
5
10
12
17
88
42
23
8
300
2
234
1
3
