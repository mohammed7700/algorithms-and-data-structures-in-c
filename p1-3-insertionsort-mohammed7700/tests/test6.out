1 2 3 4 5 8 10 12 17 23 23 42 88 98 99 234 300 
